install_debs_into_image.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+# customize raspbian image for revolution pi
+
+usage () {
+	echo 'Usage: customize_image.sh [-m, --minimize | -h, --help] <image>
+  -m, --minimize	Install only software that is necessary for basic operation (eg. Pictory and other RevPi tools)
```

Comment:
I think we can omit the whole minimg handling in this script, as it is not used at all.

Reply:
good idea, I will remove it